### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by maximilianomurua**
falta corregir cosas pero ya no me da la cabeza XD en vez de cambiar todo solo agregue div :D espero su devolucion muchacho

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,27 +91,30 @@ <h5 class="text-secondary">
       </p>
     </div>
   </div>
+<div>
   <div>
```
Comment (manutuero) [inline index.html]: Why you use an empty div here?

File: index.html [hunk 2]
```diff
@@ -22,20 +22,22 @@ <h1 class="title">
       Front-End Tools
     </h1>
   </div>
+
+ <div class="container-primary">
+     <div class="technology1">
+    <img alt="react logo" class="tech-logo1" src="../training/assets/react_icon.svg" />
```
Comment (manutuero) [inline index.html]: You are leaving the training folder (`..`) and then return `../training/assets/react_icon.svg`.
Replace your src value by `src="assets/triangle.svg"`.

File: index.html [hunk 3]
```diff
@@ -22,20 +22,22 @@ <h1 class="title">
       Front-End Tools
     </h1>
   </div>
+
+ <div class="container-primary">
+     <div class="technology1">
```
Comment (manutuero) [inline index.html]: Try to avoid numbers in the class names. Be more expressive.

File: index.html (hunk 2 above)
Comment (manutuero) [inline index.html]: same

File: index.html [hunk 4]
```diff
@@ -89,27 +91,30 @@ <h5 class="text-secondary">
       </p>
     </div>
   </div>
+<div>
   <div>
-    <img alt="title icon" class="img-title" src="../training/assets/triangle.svg" />
-    <h1 class="title">
+    <img alt="title icon" class="img-title2" src="../training/assets/triangle.svg" />
```
Comment (manutuero) [inline index.html]: same

File: index.html [hunk 5]
```diff
@@ -89,27 +91,30 @@ <h5 class="text-secondary">
...
+    <h1 class="title2">
       Back-End Tools
     </h1>
   </div>
+  
+<div class="container-secondary">
+  <div class="technology2">
```
Comment (manutuero) [inline index.html]: same

File: index.html [hunk 6]
```diff
@@ -89,27 +91,30 @@ <h5 class="text-secondary">
...
+    <img alt="rails logo" class="tech-logo1" src="../training/assets/rails_icon.png" />
```
Comment (manutuero) [inline index.html]: same

File: index.html [hunk 7]
```diff
@@ -153,14 +158,15 @@ <h5 class="text-secondary">
...
-      WE ARE GAME CHANGERS
-    </h4>
-    <img class="footer-img" alt="imagen-footer" src="../training/assets/wolox_logo_white.svg" />
-    <span class="span">
-      Copyright 2018. Wolox. all rights reserved
-    </span>
-  </div>
+</div>
+<div class="footer">
+  <h4 class="subtitle">
```
Comment (manutuero) [inline index.html]: Rename the class by `footer-subtitle`

File: index.html [hunk 8]
```diff
@@ -153,14 +158,15 @@ <h5 class="text-secondary">
...
+    WE ARE GAME CHANGERS
+  </h4>
+  <img class="footer-img" alt="imagen-footer" src="../training/assets/wolox_logo_white.svg" />
```
Comment (manutuero) [inline index.html]: Replace `alt="imagen-footer"` by `alt="footer image"`

Comment (manutuero) [inline index.html]: same

File: scss/grid.scss [hunk 9]
```diff
@@ -0,0 +1,94 @@
+.main-container {
+    display: grid;
+}
```
Comment (manutuero) [inline scss/grid.scss]: Add a empty line here

File: scss/grid.scss [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+.container-primary{
```
Comment (manutuero) [inline scss/grid.scss]: Add a space between `{` and the class name.

File: scss/grid.scss [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+  height: 601px;
+  width: 751px;
```
Comment (manutuero) [inline scss/grid.scss]: If the value of width or heigth is upper than 250px, you must use `max-heigth: 600px;` or `max-width: 750px;` props and `height: 100%;` and `width: 100%;,`

File: scss/grid.scss [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+  position: relative;
+  left: 15%;
```
Comment (manutuero) [inline scss/grid.scss]: Use `px` and not percentages.

File: scss/grid.scss [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+  bottom: 42%;
```
Comment (manutuero) [inline scss/grid.scss]: same

File: scss/grid.scss [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+}
```
Comment (manutuero) [inline scss/grid.scss]: Add an empty line here

File: scss/grid.scss [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+.container-secondary{
```
Comment (manutuero) [inline scss/grid.scss]: Add a space {

File: scss/grid.scss [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+  height: 601px;
```
Comment (manutuero) [inline scss/grid.scss]: same, use max-heigth, max-width

File: scss/grid.scss [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+  width: 751px;
+  position: relative;
+  bottom: 42%;
```
Comment (manutuero) [inline scss/grid.scss]: Same, not use percentages.

File: scss/grid.scss [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+  top: 50px;
+}
+
+.container-technology {
+  width: 413px;
```
Comment (manutuero) [inline scss/grid.scss]: use max-width

File: scss/grid.scss [hunk 19]
```diff
@@ -0,0 +1,94 @@
...
+  height: 600px;
```
Comment (manutuero) [inline scss/grid.scss]: use max-heigth

File: scss/grid.scss [hunk 20]
```diff
@@ -0,0 +1,94 @@
...
+  border-left: solid 1px;
```
Comment (manutuero) [inline scss/grid.scss]: Use a shorthand here `border: 1px solid $green;`

File: scss/grid.scss [hunk 21]
```diff
@@ -0,0 +1,94 @@
...
+  border-color: $green;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: repeat(2, 210px);
+  grid-template-rows: repeat(2, 300px);
+  position: relative;
+  left: 45%;
```
Comment (manutuero) [inline scss/grid.scss]: Avoid percentages.

File: scss/grid.scss [hunk 22]
```diff
@@ -0,0 +1,94 @@
...
+
```
Comment (manutuero) [inline scss/grid.scss]: delete this empty line

File: scss/grid.scss [hunk 23]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.technology {
+  display: grid;
+  align-content: stretch;
+  justify-items: center;
+  font-family: $roboto;
+}
+
```
Comment (manutuero) [inline scss/grid.scss]: delete this emtpy line

File: scss/grid.scss [hunk 24]
```diff
@@ -0,0 +1,94 @@
...
+
+.tech-logo {
+  height: $tech-logo-size;
+  width: $tech-logo-size;
+  display: grid;
+  justify-content: center;
+}
+
+.tech-logo1 {
```
Comment (manutuero) [inline scss/grid.scss]: avoid numbers in class names

File: scss/grid.scss [hunk 25]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.text-tertiary {
+  font-size: $text-small;
+  text-align: center;
+  padding: 10px 24px 0;
+  text-align: center;
+}
+
+.technology1{
```
Comment (manutuero) [inline scss/grid.scss]: same

File: scss/grid.scss [hunk 26]
```diff
@@ -0,0 +1,94 @@
...
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: repeat(2, 210px);
+  grid-template-rows: repeat(2, 300px);
+  position: relative;
+  left: 45%;
+
+}
+
+.technology {
```
Comment (manutuero) [inline scss/grid.scss]: Why you define the `technology` styles in the `grid.scss` file?? :thinking: Move this classes to the technology.scss file

File: scss/title.scss [hunk 27]
```diff
@@ -2,16 +2,37 @@ $title-decoration-height: 90px;
...
 
-.img {
+.img-title {
   height: $title-decoration-height;
   width: $title-decoration-width;
+  margin-top: 50px; 
+  margin-left: 70px;
 }
 
+.img-title2 {
```
Comment (manutuero) [inline scss/title.scss]: same

File: scss/technology.scss [hunk 28]
```diff
@@ -1,77 +0,0 @@
-.img-title {
```
Comment (manutuero) [inline scss/technology.scss]: Why you delete this file?

File: scss/title.scss [hunk 29]
```diff
@@ -2,16 +2,37 @@ $title-decoration-height: 90px;
 $logo-decoration-height: 100px;
 $title-decoration-width: 70px;
 
-.img {
+.img-title {
   height: $title-decoration-height;
   width: $title-decoration-width;
+  margin-top: 50px;
```
Comment (manutuero) [inline scss/title.scss]: Use a shorthand here

File: scss/title.scss [hunk 30]
```diff
@@ -2,16 +2,37 @@ $title-decoration-height: 90px;
...
   width: $title-decoration-width;
+  margin-top: 50px; 
+  margin-left: 70px;
 }
 
+.img-title2 {
+  height: $title-decoration-height;
+  width: $title-decoration-width;
+  position: relative;
+  right: 16%;
```
Comment (manutuero) [inline scss/title.scss]: same

File: scss/title.scss [hunk 31]
```diff
@@ -2,16 +2,37 @@ $title-decoration-height: 90px;
...
+}
+
+.title2 {
+  margin: 200px 0 0;
+  font-weight: bold;
+  font-size: $text-big;
+  left: -24px;
+  position: relative;
+  right: 19%;
```
Comment (manutuero) [inline scss/title.scss]: same

File: index.html (hunk 1 above)
Comment (maximilianomurua) [reply index.html]: As I did not want to disarm everything, add div.
first div - encloses the primary container
second div - secondary container.
third div - belongs to the polymer logo technology

File: scss/technology.scss (hunk 28 above)
Comment (maximilianomurua) [reply scss/technology.scss]: why I take it out of technology and pass it to the title.

File: index.html [hunk 32]
```diff
@@ -158,15 +152,15 @@ <h5 class="text-secondary">
...
-  <h4 class="subtitle">
-    WE ARE GAME CHANGERS
-  </h4>
-  <img class="footer-img" alt="imagen-footer" src="../training/assets/wolox_logo_white.svg" />
-  <span class="span">
-    Copyright 2018. Wolox. all rights reserved
-  </span>
-</div>
+  <div class="footer">
+    <h4 class="subtitle">
```
Comment (manutuero) [inline index.html]: Change class name by footer-title

File: index.html [hunk 33]
```diff
@@ -158,15 +152,15 @@ <h5 class="text-secondary">
...
+      WE ARE GAME CHANGERS
+    </h4>
+    <img class="footer-img" alt="imagen-footer" src="assets/wolox_logo_white.svg" />
+    <span class="span">
```
Comment (manutuero) [inline index.html]: Rename class name by copyright

File: index.html [hunk 34]
```diff
@@ -158,15 +152,15 @@ <h5 class="text-secondary">
...
-<div class="footer">
-  <h4 class="subtitle">
-    WE ARE GAME CHANGERS
-  </h4>
-  <img class="footer-img" alt="imagen-footer" src="../training/assets/wolox_logo_white.svg" />
-  <span class="span">
-    Copyright 2018. Wolox. all rights reserved
-  </span>
-</div>
+  <div class="footer">
```
Comment (manutuero) [inline index.html]: You can use the html5 tag "footer"

File: index.html [hunk 35]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (manutuero) [inline index.html]: Remove empty line..configure your vscode formatter for avoid this

File: index.html [hunk 36]
```diff
@@ -11,35 +12,32 @@
...
-    </h5>
-    <p class="text-tertiary">
-      React is a declarative, component-based Javascript library for
-      building user interfaces.
-    </p>
-     </div>
   <div class="container-technology">
+    <div class="technology item">
+      <img alt="react logo" class="tech-logo" src="assets/react_icon.svg" />
+      <h4 class="text-primary">
```
Comment (manutuero) [inline index.html]: Rename these class names to be more expressive.

File: scss/grid.scss (hunk 20 above)
Comment (manutuero) [reply scss/grid.scss]: Use shorthand!

File: scss/grid.scss [hunk 37]
```diff
@@ -1,94 +1,15 @@
...
-  height: 250px;
-  width: 189px;
-  position: absolute;
-  top: 30%;
-  left: 20%;
+  grid-template-columns: 240px 230px 200px;
+  grid-template-rows: 315px 300px;
 }
 
+.item {
```
Comment (manutuero) [inline scss/grid.scss]: Rename this class name to be more expressive.

File: scss/technology.scss [hunk 38]
```diff
@@ -0,0 +1,36 @@
...
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
+
+.technology {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  font-family: $roboto;
+  max-height: 100%;
```
Comment (manutuero) [inline scss/technology.scss]: max-heigth uses a value not percentage.

File: scss/technology.scss [hunk 39]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.tech-logo {
+  height: $tech-logo-size;
+  width: $tech-logo-size;
+}
+
+.text-primary {
```
Comment (manutuero) [inline scss/technology.scss]: Rename these css class names

File: scss/title.scss [hunk 40]
```diff
@@ -2,37 +2,19 @@ $title-decoration-height: 90px;
...
-  left: -24px;
   position: relative;
-  top: -30px;
+
+  &::before {
+    content: "";
+    background: url("assets/triangle.svg") no-repeat;
+    height: 90px;
+    position: absolute;
+    left: -50px;
```
Comment (manutuero) [inline scss/title.scss]: I think that you can avoid negative values. Please try to do this with margins.

Comment (maximilianomurua) [reply scss/title.scss]: you can not use margin.

**Pull request merged**